db/memtable.cc

```diff
@@ -412,13 +412,28 @@ InternalIterator* MemTable::NewIterator(const ReadOptions& read_options,
   return new (mem) MemTableIterator(*this, read_options, arena);
 }
 
-InternalIterator* MemTable::NewRangeTombstoneIterator(
+FragmentedRangeTombstoneIterator* MemTable::NewRangeTombstoneIterator(
```

Comment:
Thanks for making the contracts of MemTable::NewRangeTombstoneIterator and BlockBasedTable::NewRangeTombstoneIterator consistent!